spec/release/notes/system_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe Release::Notes::System do
+  let(:klass) { Release::Notes::System }
+
+  describe "#all_tags" do
+    it "calls Git.read_all_tags" do
+      expect(Release::Notes::Git).to receive(:read_all_tags).and_return("foo")
+      allow(Release::Notes::System).to receive(:`)
+      klass.all_tags
+    end
+
+    it "hits the cmd line" do
+      allow(Release::Notes::Git).to receive(:read_all_tags).and_return("foobar")
+      expect(Release::Notes::System).to receive(:`).with("foobar").once
+      klass.all_tags
+    end
+
+    it "returns the result of the system program" do
+      allow(Release::Notes::System).to receive(:`).and_return("foo")
+      expect(klass.all_tags).to eq "foo"
+    end
+  end
+
+  describe "#system_last_tag" do
+    it "calls Git.system_last_tag" do
+      expect(Release::Notes::Git).to receive(:last_tag).and_return("foo")
+      allow(Release::Notes::System).to receive(:`)
+      klass.system_last_tag
+    end
+
+    it "hits the cmd line" do
+      allow(Release::Notes::Git).to receive(:last_tag).and_return("foobar")
+      expect(Release::Notes::System).to receive(:`).with("foobar").once
+      klass.system_last_tag
+    end
+
+    it "returns the result of the system program" do
+      allow(Release::Notes::System).to receive(:`).and_return("foo")
+      expect(klass.system_last_tag).to eq "foo"
+    end
+  end
+
+  describe "#tag_date" do
+    it "calls Git.tag_date" do
+      expect(Release::Notes::Git).to receive(:tag_date).and_return("foo")
+      allow(Release::Notes::System).to receive(:`)
+      klass.tag_date
+    end
+
+    # it "hits the cmd line" do
```

Comment:
Let's make this test work or just remove - no need for commenting out here.

Reply:
👍